Client/Frontend/Widgets/ActionOverlayTableViewController.swift

```diff
@@ -24,9 +26,11 @@ class ActionOverlayTableViewController: UIViewController, UITableViewDelegate, U
         return visualEffectView
     }()
 
-    init(site: Site, actions: [ActionOverlayTableViewAction]) {
+    init(site: Site, actions: [ActionOverlayTableViewAction], headerIconHandler: () -> UIImage?, headerIconBackgroundHandler: () -> UIColor?) {
```

Comment:
There is no need to use closures here. You can simply just pass the image/color directly to ActionOverlayTableViewController